src/sentry/integrations/issues.py

```diff
@@ -34,6 +34,15 @@ def get_group_description(self, group, event, **kwargs):
         return '\n'.join(output)
 
     def get_create_issue_config(self, group, **kwargs):
+        """
+        These fields are used to render a form for the user,
+        and are then passed in the format of:
+
+        >>>{'title': 'TypeError: Object [object Object] has no method "updateFrom"''}
```

Comment:
Hmm should this have both `title` and `description` since that is what this function returns?

Reply:
i was trying to show the format that the UI eventually sends back rather than what the function returns